<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgSParam</class>
 <widget class="QDialog" name="CfgSParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditFilePath"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolopenFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>脚本文件夹：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabACC">
      <attribute name="title">
       <string>ACC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupACC">
         <property name="title">
          <string>休眠后测试：</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>循环次数：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinAccCycle">
            <property name="toolTip">
             <string>1~65535</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
            <item>
             <widget class="QTreeWidget" name="treeACC">
              <attribute name="headerDefaultSectionSize">
               <number>150</number>
              </attribute>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
              <column>
               <property name="text">
                <string>参数</string>
               </property>
              </column>
              <item>
               <property name="text">
                <string>keyACC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>关ACC</string>
               </property>
               <item>
                <property name="text">
                 <string>时间(mS)：</string>
                </property>
                <property name="toolTip">
                 <string>关处理时间</string>
                </property>
                <property name="text">
                 <string>10000</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>时间变动</string>
                </property>
                <property name="toolTip">
                 <string>休眠时间设置为可变处理；</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </property>
                <property name="checkState">
                 <enum>Unchecked</enum>
                </property>
                <item>
                 <property name="text">
                  <string>递增：</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>min(mS):</string>
                 </property>
                 <property name="text">
                  <string>6000</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>max(mS):</string>
                 </property>
                 <property name="text">
                  <string>30000</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>step(mS):</string>
                 </property>
                 <property name="text">
                  <string>50</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
               </item>
               <item>
                <property name="text">
                 <string>休眠电流(check)</string>
                </property>
                <property name="toolTip">
                 <string>关机时间超过休眠电流检测时间将检测休眠电流；</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </property>
                <property name="checkState">
                 <enum>Checked</enum>
                </property>
                <item>
                 <property name="text">
                  <string>时间(mS):</string>
                 </property>
                 <property name="text">
                  <string>15000</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>电流(mA):</string>
                 </property>
                 <property name="text">
                  <string>20</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
               </item>
              </item>
              <item>
               <property name="text">
                <string>开ACC</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <item>
                <property name="text">
                 <string>时间(mS):</string>
                </property>
                <property name="toolTip">
                 <string>开处理时间</string>
                </property>
                <property name="text">
                 <string>1000</string>
                </property>
                <property name="toolTip">
                 <string/>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>唤醒电流(check)</string>
                </property>
                <property name="toolTip">
                 <string>工作电流是否达到正常工作电流</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="checkState">
                 <enum>Checked</enum>
                </property>
                <item>
                 <property name="text">
                  <string>电流(mA)：</string>
                 </property>
                 <property name="text">
                  <string>320</string>
                 </property>
                 <property name="toolTip">
                  <string>电流最好为工作电流减去10~50mA</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
               </item>
               <item>
                <property name="text">
                 <string>记忆检测(check)</string>
                </property>
                <property name="toolTip">
                 <string>检测开机后是否回到上次关机前界面</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="toolTip">
                 <string>检测是否回到休眠前界面，反之检测存在界面即OK</string>
                </property>
                <property name="checkState">
                 <enum>Checked</enum>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>超时时间(mS)：</string>
                </property>
                <property name="toolTip">
                 <string>开机测试检测最长时间，超过该时间将不再检测，判断为失败。</string>
                </property>
                <property name="text">
                 <string>60000</string>
                </property>
                <property name="toolTip">
                 <string>处理超过该时间将结束测试，并判断为失败</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
              </item>
              <item>
               <property name="text">
                <string>script</string>
               </property>
               <item>
                <property name="text">
                 <string>logcheck:</string>
                </property>
                <property name="text">
                 <string>OK (1 test)</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textACCDescription">
              <property name="toolTip">
               <string>测试单元描述</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBAT">
      <attribute name="title">
       <string>B+</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBAT">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>关机后测试：</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>循环次数：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBatCycle">
            <property name="toolTip">
             <string>1~65535</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
            <item>
             <widget class="QTreeWidget" name="treeBAT">
              <attribute name="headerDefaultSectionSize">
               <number>150</number>
              </attribute>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
              <column>
               <property name="text">
                <string>参数</string>
               </property>
              </column>
              <item>
               <property name="text">
                <string>KeyBAT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>关B+</string>
               </property>
               <item>
                <property name="text">
                 <string>时间(mS):</string>
                </property>
                <property name="text">
                 <string>5000</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
              </item>
              <item>
               <property name="text">
                <string>开B+</string>
               </property>
               <item>
                <property name="text">
                 <string>时间(mS):</string>
                </property>
                <property name="text">
                 <string>1000</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>开机电流</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="checkState">
                 <enum>Checked</enum>
                </property>
                <item>
                 <property name="text">
                  <string>电流(mA):</string>
                 </property>
                 <property name="text">
                  <string>320</string>
                 </property>
                 <property name="toolTip">
                  <string>电流最好为工作电流减10~50</string>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
               </item>
               <item>
                <property name="text">
                 <string>记忆检测</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="checkState">
                 <enum>Checked</enum>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>超时时间(mS):</string>
                </property>
                <property name="text">
                 <string>60000</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
              </item>
              <item>
               <property name="text">
                <string>script</string>
               </property>
               <item>
                <property name="text">
                 <string>logcheck</string>
                </property>
                <property name="text">
                 <string>OK (1 test)</string>
                </property>
                <property name="flags">
                 <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                </property>
               </item>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textBatDescription"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CfgSParam</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CfgSParam</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
